app/views/user/new.html.haml

```diff
@@ -0,0 +1,30 @@
+%h1 New User
+%br/
+= form_for @user, url: '/user', html: { method: :post } do |f|
```

Comment:
Check the layout of the form of the theme
http://responsiweb.com/themes/preview/ace/1.3.3/form-elements.html

Your form has to follows standard of bootstrap